### ISSUE TITLE
Feature request: search dropdown items

### ISSUE THREAD

**Issue by taeh98** [react-native-picker/picker#249]
Hello, I hope this finds you well.

Please could you allow adding searchbars to the top of dropdown pickers that would allow users to type text to search for a corresponding dropdown item? This would allow users to find dropdown items more quickly in certain applications. A similar feature exists in https://github.com/hossein-zare/react-native-dropdown-picker

Thank you for your time and consideration.

**Comment by rajinpangkalpundai** [comment]
Yeah, this would be helpful if we have a long list of dropdown item, any news on this guys?

**Comment by sajeduardo** [comment]
I have a Dropdown with more then 300 items, so if we have an searchable possibility will be good.

**Comment by Julio-de-Leon** [comment]
Any update on this feature?

**Comment by Ali-Aref** [comment]
Same here. This issue is open from **2021** so basically I think I should go for other package.
